pom.xml

```diff
@@ -5,6 +5,30 @@
   <groupId>org.profit.telegrambot</groupId>
   <artifactId>telegram-test-bot</artifactId>
   <version>1.0-SNAPSHOT</version>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>14</source>
+          <target>14</target>
+        </configuration>
+      </plugin>
+    </plugins>
```

Comment:
You can shrink these lines up to only two ones nested in `<properties>` section: 

- `<maven.compiler.source>`
- `<maven.compiler.target>`

PS: why exactly 14?)